pkg/controller/statefulset/stateful_set.go

```diff
@@ -143,8 +143,8 @@ func (ssc *StatefulSetController) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer ssc.queue.ShutDown()
 
-	glog.Infof("Starting stateful set controller")
-	defer glog.Infof("Shutting down statefulset controller")
+	glog.Info("Starting stateful set controller")
```

Comment:
While you're at it, could you fix the text to be consistent for these two lines? Let's call it "StatefulSet controller".

Reply:
ok,thanks,I have fixed it,for your review!@enisoc